builds.yml

```
@@ -45,6 +48,7 @@ buildTypes:
     isPrerelease: true
     # Folder which contains overrides to browser manifests
     manifestOverrides: ./app/build-types/beta/manifest/
+    buildNameOverride: false
```

Maybe we should just write in the build name for beta and Flask, just for explicitness:
```suggestion
    buildNameOverride: MetaMask Beta
```